### ISSUE TITLE
Regression/correlation threats nominal variables as scale variables

### ISSUE THREAD

**Issue by Ahjvdberg** [jasp-stats/jasp-issues#329]
When using jasp for correlation analysis, it also calculates the correlation for nominal variables (if these are represented by numeric values). For nominal variables a correlation should not be possible! The same happens when doing regression, it ignores the variable's classification as nominal and threats it as if it is a scale variable. I'm using JASP version 0.9.2.0. 
This is a very serious issue and can very easily turn into incorrect results!

**Comment by EJWagenmakers** [comment]
Yes, we should not allow nominal variables to be entered for this analysis
@boutinb can you assign this to someone?

**Comment by FransMeerhoff** [comment]
Hi @TimKDJ  @vandenman 

Can one of you please have a look at this?

**Comment by AlexanderLyNL** [comment]
This is an ongoing discussion, but when someone adds variables into a linear regression I think that it's fine for the variables to be interpreted as scale, otherwise they should do an ancova instead. I don't think that we should change this behaviour.

**Comment by EJWagenmakers** [comment]
I am sticking to my position. Lexi, let's discuss this in person.

**Comment by tomtomme** [comment]
This is partly fixed for regression. I can not enter nominal Covariates, but I can enter a nominal DV.
Also I can still add nominal variables in the correlation module.

Both produces this error message. 

```
This analysis terminated unexpectedly.

Error: Trying to decode columnName but 'attract' is not an encoded columnName!

Stack trace

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports
```

would be better to just not allow entering nominals.

**Comment by tomtomme** [comment]
@boutinb 
This is a very old bug, someone should be assigned, yes?

**Comment by tomtomme** [comment]
@Ahjvdberg 
I think this fixed now with 0.19 beta

![grafik](https://github.com/jasp-stats/jasp-issues/assets/4428185/fd02a119-c01c-4772-b020-93759eef03b8)

You see here, that JASP now informs the user which scales are okay and which are technically possible, but should probably not be used (more transparent icons)
Please reopen if you think, that this does not solve the issue!!

To test yourself you could get the latest beta from here: https://static.jasp-stats.org/Nightlies/
